docs/application/tizen-studio/native-tools/manifest-text-editor.md

```diff
@@ -1210,6 +1213,65 @@ For more information on the relationship between the elements, see the [element
 <feature name="http://tizen.org/feature/fmradio">true</feature>
 ```
 
+### \<dependencies\> Element
+
+This element contains a set of packages which are have dependent to this Tizen package. The package can be installed when all dependencies are satisfied.
```

Comment:
This is sentence is confusing for an end-user to read. Do you mean:

This element contains a set of packages that have dependencies on the Tizen package. The package can be installed when all dependencies are satisfied.

Kindly clarify so we can rephrase it better.

Reply:
Yes. I fixed it. sorry for my typo.